modules/ocf_tv/files/mpd.conf

```diff
@@ -0,0 +1,17 @@
+music_directory     "/var/lib/mpd/music"
+playlist_directory  "/var/lib/mpd/playlists"
+db_file             "/var/lib/mpd/tag_cache"
+log_file            "/var/log/mpd/mpd.log"
+pid_file            "/run/mpd/pid"
+state_file          "/var/lib/mpd/state"
+sticker_file        "/var/lib/mpd/sticker.sql"
+user                "mpd"
+bind_to_address     "any"
+
+input {
+  plugin            "curl"
+}
+audio_output {
+  type              "pulse"
+  name              "Sink #1"
+}
```

Comment:
I wonder if the encoding field should still be kept so that it’s ensured it is operating in UTF-8